test/controllers/due.diligence.controller.spec.ts

```diff
@@ -1,3 +1,5 @@
+import * as config from "../../src/config";
```

Comment:
we already have an import from ../../src/config, can you merge the 2 together

Reply:
The two are now merged.